lambdas/models/pds_models.py

```diff
@@ -66,10 +66,10 @@ class Patient(BaseModel):
 
     id: str
     birth_date: date
-    address: Optional[list[Address]] = []
+    address: Optional[list[Address]] = None
```

Comment:
Agree to use None here as the default, but we may need to amend the method `get_current_home_address` at the same time, as the line `for entry in self.address:` assumes self.address to be an array.

Reply:
Missed that one, thanks!